src/nbody_simulation_result.jl

```diff
@@ -0,0 +1,172 @@
+# SimulationResult sould provide an interface for working with properties of a separate particle
+# and with physical properties of the whole system.
+struct SimulationResult{sType <: NBodySystem}
+    solution::AbstractTimeseriesSolution
+    simulation::NBodySimulation{sType}
+end
+
+function Base.show(stream::IO, sr::SimulationResult)
+    print(stream, "N: ") 
+    show(stream, length(sr.simulation.system.bodies))
+    println(stream)
+    show(stream, sr.simulation.system)
+    print(stream, "Time steps: ") 
+    show(stream, length(sr.solution.t))
+    println(stream)
+    print(stream, "t: ") 
+    show(stream, sr.solution.t)
+end
+
+(sr::SimulationResult)(t) = return sr.solution(t)
```

Comment:
there's in-place forms and other options to split. I'd just `args...;kwargs...` splat the whole thing through

Reply:
Oh, yes. Thanks. I somehow thought about indexing by t here. Surely it is an argument for the function.